extension/content.js

```diff
@@ -26,6 +28,28 @@ function linkifyBranchRefs() {
 	});
 }
 
+function addReleasesTab() {
+	const releasesTabTemplate = '<span itemscope="" itemtype="http://schema.org/ListItem" itemprop="itemListElement">' +
```

Comment:
We don't need this `itemscope="" itemtype="http://schema.org/ListItem" itemprop="itemListElement"`